src/PseudoArcLengthContinuation.jl

```diff
@@ -86,11 +86,11 @@ module PseudoArcLengthContinuation
 						tangentalgo = SecantPred(),
 						linearalgo  = BorderingBLS(),
 						plot = false,
-						printsolution = norm,
+						printsolution = u -> u,
 						normC = norm,
 						plotsolution = (x;kwargs...) -> nothing,
 						finaliseSolution = (z, tau, step, contResult) -> true,
-						verbosity = 2) where {T, S <: AbstractLinearSolver, E <: EigenSolver}
+						verbosity = 0) where {T, S <: AbstractLinearSolver, E <: EigenSolver}
```

Comment:
changed the default arguments to output all components and verbosity=0 to avoid StackOverflow during `Flux.train!` Furthermore a strange `illegal instruction / unknown function` crash occurs when
```julia
g() = Cont.continuation( f, u₀,p₀, ContinuationPar() )
f() = Cont.continuation( f, u₀,p₀, ContinuationPar(); verbosity = 0, printsolution = u -> u )
g() # success
f() # julia crashes
```

Reply:
I saw that. I answered this. I can push those changes but I feel you should get a MWE using TrackedArrays. The changes below that you propose go the same way. Don't you think?

Reply:
I created a branch with the example you requested https://github.com/gszep/FluxContinuation/tree/minimal-working-example
it successfully executes for me. let me know !